patches/minecraft/net/minecraft/entity/Entity.java.patch

```diff
@@ -1,6 +1,14 @@
 --- ../src-base/minecraft/net/minecraft/entity/Entity.java
 +++ ../src-work/minecraft/net/minecraft/entity/Entity.java
-@@ -84,7 +84,7 @@
+@@ -10,6 +10,7 @@
+ import java.util.Random;
+ import java.util.Set;
+ import java.util.UUID;
++import javax.annotation.Nonnull;
```

Comment:
Avoid adding this import by using `@javax.annotation.Nonnull` where you need it.